app/api/settings/specs/settings.spec.js

```diff
@@ -190,6 +190,26 @@ describe('settings', () => {
       const values = await settings.get();
       expect(values.publicFormDestination).not.toBeDefined();
     });
+
+    it('should return default values', async () => {
+      const values = await settings.get();
+      expect(values.mapTilerKey).toEqual('QiI1BlAJNMmZagsX5qp7');
+    });
+
+    describe('if there is settings with no default mapTilerKey on the DB', () => {
+      it('should return the stored mapTilerKey', done => {
```

Comment:
We have moved away from the (done) + Promises approach, and now use the async-await model.  Please refactor this to use that, you will see how much 'cleaner' and readable it looks.

Reply:
👍 you're right the code is better now